runHarness/WorkloadDrivers/AuctionWorkloadDriver.pm

```diff
@@ -1174,19 +1186,87 @@ sub startRun {
 	my $endRunStatus = "";
 	my $endRunStatusRaw = "";
 	$url = $self->getControllerURL() . "/run/$runName/state";
+	my @curIntervalNames;
+	my @lastIntervalNames;
+	my @statsRunning;
+
 	while (!$runCompleted) {
 		$logger->debug("Sending get to $url");
 		$req = HTTP::Request->new( GET => $url );
 		$res = $ua->request($req);
 		$logger->debug(
 			"Response status line: " . $res->status_line . " for url " . $url );
 		if ( $res->is_success ) {
-			$endRunStatus = $json->decode( $res->content );	
+			$logger->debug("Response content: " . $res->content );
```

Comment:
Can you should take this out.  I had put this when I was looking at the returned content, but it really fills up the debug log.